src/System.ObjectModel/src/System/Collections/ObjectModel/ObservableCollection.cs

```diff
@@ -46,6 +46,22 @@ public class ObservableCollection<T> : Collection<T>, INotifyCollectionChanged,
         /// <exception cref="ArgumentNullException"> collection is a null reference </exception>
         public ObservableCollection(IEnumerable<T> collection) : base(CreateCopy(collection)) { }
 
+        /// <summary>
+        /// Initializes a new instance of the ObservableCollection class
+        /// that contains elements copied from the specified list
+        /// </summary>
+        /// <param name="list">The list whose elements are copied to the new list.</param>
+        /// <remarks>
+        /// The elements are copied onto the ObservableCollection in the
+        /// same order they are read by the enumerator of the list.
+        /// </remarks>
+        /// <exception cref="ArgumentNullException"> list is a null reference </exception>
+        public ObservableCollection(List<T> list)
+            : base((list != null) ? new List<T>(list.Count) : list)
+        {
+            CreateCopy(list);
+        }
```

Comment:
I would have expected that test to fail with the initial PR changes. I guess it didn't fail because the tests aren't running against the newly exposed surface area?

BTW, I'd also expect [this test](https://github.com/dotnet/corefx/blob/10580bee652ca12c3bf85f11e8b2f6ce2ade93af/src/System.ObjectModel/tests/ObservableCollection/ObservableCollection_ConstructorAndPropertyTests.cs#L72) to fail during compilation due to ambiguity between the two constructors when passing `null`.

Ideally, there would be a `ObservableCollectionTests.netstandard1.7.cs` that would test the newly exposed constructor, ensuring it copies the passed-in `List<T>` and throws `ArgumentNullException` with the `list` paramName.